pkg/reconciler/v1alpha1/service/service.go

```diff
@@ -163,34 +162,65 @@ func (c *Reconciler) reconcile(ctx context.Context, service *v1alpha1.Service) e
 
 	service.Status.InitializeConditions()
 
+	config, err := c.config(ctx, service)
```

Comment:
I'd call this something like reconcileConfiguration, but I'd also fold the status manipulation into this.

Reply:
There is already a reconcileConfiguration function in this file. Changed to initalizeService as it now contains service initialization calls.